### PR TITLE
Add language selection

### PR DESCRIPTION
**Pull request by Ekristoffe**
From a clean base this time

### CODE REVIEW

File: alvr/common/src/data/session.rs [hunk 1]
```diff
@@ -112,6 +112,7 @@ pub struct ClientConnectionDesc {
 #[serde(rename_all = "camelCase")]
 pub struct SessionDesc {
     pub setup_wizard: bool,
+    pub i18n_locale: String,
```
Comment (zarik5) [inline alvr/common/src/data/session.rs:115]: I would like to keep names not linked to specific libraries. "locale" should be enough.

**Review by zarik5 — CHANGES_REQUESTED** [review]
Thanks. Sometimes when selecting the language the page refreshes but the language does not change. Selecting the language again then works. We can fix this another time.

File: alvr/common/src/data/session.rs (hunk 1 above)
Comment (Ekristoffe) [reply alvr/common/src/data/session.rs]: Naming changed

**Comment by Ekristoffe** [comment]
> 
> 
> Thanks. Sometimes when selecting the language the page refreshes but the language does not change. Selecting the language again then works. We can fix this another time.

Do you know which language does this ?
In the console is there a message ?

**Pull request merged**